<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsWidget</class>
 <widget class="QWidget" name="ItemsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="_3">
     <item row="0" column="0">
      <widget class="QLineEdit" name="SearchLine_"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="SearchType_">
       <item>
        <property name="text">
         <string>Fixed string</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wildcard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regexp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Important (this channel)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="CaseSensitiveSearch_">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="CategoriesSplitter_">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="Items_">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="LC::Util::SelectableBrowser" name="ItemView_" native="true"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC::Util::SelectableBrowser</class>
   <extends>QWidget</extends>
   <header location="global">util/gui/selectablebrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
